config/locales/actions.en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  actions:
+    archive_prompt:
+      label: "Archive"
+      confirmation: "Are you sure you want to archive %{name}?"
```

Comment:
`"Are you sure you want to archive ""%{name}?"""`

I believe this should work to produce the following (what I would like)

>Are you sure you want to archive "Name"?